dask/dataframe/indexing.py

```diff
@@ -2,10 +2,162 @@
 
 from datetime import datetime
 
+from toolz import merge
 import bisect
 import numpy as np
 import pandas as pd
 
+from .core import new_dd_object, Series
+from . import methods
+from ..base import tokenize
+
+
+class _LocIndexer(object):
+    """ Helper class for the .loc accessor """
+
+    def __init__(self, obj):
+        self.obj = obj
+
+    @property
+    def _name(self):
+        return self.obj._name
+
+    def _make_meta(self, iindexer, cindexer):
+        """
+        get metadata
+        """
+        if cindexer is None:
+            return self.obj
+        else:
+            return self.obj._meta.loc[:, cindexer]
+
+    def __getitem__(self, key):
+
+        if isinstance(key, tuple):
+            # multi-dimensional selection
+            if len(key) > self.obj.ndim:
+                raise KeyError(key)
```

Comment:
Is this like

```python
In [13]: df = pd.DataFrame(np.random.randn(10, 2))

In [14]: df.loc[(0, 1, 'c')]
```

Would we want to raise an `IndexingError` here to be consistent with pandas?